VERSION.txt

```diff
@@ -1 +1 @@
-8.7.0
+8.7.1
```

Comment:
This is picky, but the minor version is the middle one, so we should have `8.8.0`.

The 'patch' version (the last one) is for bugfixes, this is kinda more than just a bugfix.